### DISCUSSION TITLE
Different offset required for lazer

### DISCUSSION THREAD

**Discussion by rodrigopina360** [ppy/osu#27256] [Q&A] [answered]
I've played lazer for quite a while, without much issue, but as of the latest release, an issue I've had is that on most maps I play I always start slightly later and hit 100s, which didn't use to be a problem, and currently isn't a problem with stable. I'd just want to know if it's something that's known already, if it might be a problem with the current release, or just something on my end.

**Comment by JacksonChen666** [comment]
osu!lazer penalizes for hitting the slider head early or late, unlike osu!stable, which doesn't do that.

https://youtube.com/watch?v=xTRwM3zhhj0&t=243

**Reply by rodrigopina360** [reply]
Yeah, I'm aware, but it's not the first slider, it's usually the first couple seconds, until I'm used to it and play properly for the rest of the map. Kinda my bad for explaining it in a way that suggest I'm "new to lazer's mechanics", but thanks for the suggestion

**Comment by OwenCMYK** [comment]
It might be because lazer has a lower input latency when compared with stable, but generally I'd recommend playing a bunch of different beatmaps and then going to the settings and clicking "apply suggested offset". It should fix your problem automatically

**Reply by rodrigopina360** [reply]
Well yeah, it could be the case, but I'd guess with lower input delay I'd be pressing circles ahead of time.. Although I have set a 5ms offset to bandaid fix it, the global suggested offset is also a nice tool I wasn't aware of. Thanks <3

**Comment by rodrigopina360** [comment, marked as answer]
Upon further testing, I have 100% noticed a difference between the average hit error in stable and in lazer, and in order to hopefully pinpoint it I downgraded to 2024.131.0 to test if the issue was still present, but the timing seemed fine. After updating again to the latest release, the issue seems to be somehow fixed, not having the inputs be late when hits feeling good. Thanks for the suggestions!
